Resources/Maps/Nonstations/clockwork-arena.yml

```
@@ -0,0 +1,2233 @@
+meta:
```

yamllint

[error] wrong new line character: expected \n

(new-lines)